workspaces/kener/plugins/kener-status/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@backstage/plugin-kener-status",
+  "version": "0.1.0",
```

Comment:
When this gets merged and published to NPM it will get increments to `0.2.0` is that what you want?